Equilobe.DailyReport.Models/Dashboard/DashboardData.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+
+namespace Equilobe.DailyReport.Models.Dashboard
+{
+    public class DashboardData
+    {
+        public bool IsAvailable { get; set; }
+
+        public List<DashboardItem> Items { get; set; }
+
+
+        public static DashboardData Unavailable
```

Comment:
make this a method instead of a property. not all objects of type DashboardData have an invalid DashboardData property on them. 
Instead, just make this as a method that returns the same thing. Also keep the name.